src/main/java/uk/gov/pay/ledger/queue/managed/QueueMessageReceiver.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.pay.ledger.queue.managed;
+
+import com.google.inject.Inject;
+import io.dropwizard.lifecycle.Managed;
+import io.dropwizard.setup.Environment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.ledger.app.LedgerConfig;
+import uk.gov.pay.ledger.app.config.QueueMessageReceiverConfig;
+import uk.gov.pay.ledger.queue.EventMessageHandler;
+import uk.gov.pay.ledger.queue.QueueException;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class QueueMessageReceiver implements Managed {
+
+    private static final String QUEUE_MESSAGE_RECEIVER_THREAD_NAME = "queue-message-receiver";
```

Comment:
Suggest including the thread id in the name eg.

```suggestion
    private static final String QUEUE_MESSAGE_RECEIVER_THREAD_NAME = "queue-message-receiver-%d";
```
see https://github.com/google/guava/blob/master/guava/src/com/google/common/util/concurrent/ThreadFactoryBuilder.java#L62-L66